src/services/base/BaseService.js

```diff
@@ -51,4 +51,12 @@ export class BaseService {
     __UseCollection(collectionName) {
         this.db = firebase.firestore().collection(collectionName)
     }
+
+    async __WARNING_CLEAR_COLLECTION() {
```

Comment:
This has been removed in master. Please make sure that you have merged in master before submitting this PR